buildSrc/src/main/kotlin/io/spine/internal/dependency/ProtoData.kt

```diff
@@ -33,7 +33,8 @@ package io.spine.internal.dependency
  */
 @Suppress("unused", "ConstPropertyName")
 object ProtoData {
-    const val version = "0.9.11"
+    const val version = "0.11.6"
+    const val dogfoodingVersion = "0.11.6"
```

Comment:
Not sure if we need two versions of ProtoData in `core-java`.

I introduced `dogfoodingVersion` in `mc-java` to separate between the version we apply to build `mc-java` — the dogfooding version — and the version we apply to the `mc-java`'s client projects — the regular version.

Reply:
I second this comment.

Reply:
Turns out this is just what `config` brings now. I guess, better have it and not use it that not have it and need it.